server/sonar-server/src/main/java/org/sonar/server/settings/ws/SetAction.java

```diff
@@ -203,6 +292,22 @@ private static SetRequest toWsRequest(Request request) {
     return setRequest;
   }
 
+  private static List<Map<String, String>> readFieldsValues(Request request) {
+    Type type = new TypeToken<Map<String, String>>() {
+    }.getType();
+    Gson gson = GsonHelper.create();
+
+    return request.multiParam(PARAM_FIELDS_VALUES).stream()
+      .map(jsonString -> {
+        try {
+          return (Map<String, String>) gson.fromJson(jsonString, type);
+        } catch (JsonParseException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)